src/desktop/components/main_layout/stylesheets/body.styl

```diff
@@ -186,3 +186,13 @@ html[data-useragent*="iPad"]
 .body-no-padding
   #main-layout-container
     padding 0
+
+// This is where reaction-built apps mount to
+#react-root
+  position: relative;
+
+  .reactionPageLoader
+    width: 100%;
+    position: fixed;
+    left: 0;
+    top: 53px !important; // Overwrite storybooks position
```

Comment:
Don't like the `important` but since it's impossible to use `position: fixed` in combination with `position: relative` on the parent, we need to set a fixed position in force and then a separate position in Reaction, as we're not within the page shell.